alphagov/paas-team-manual
# Update ZenDesk tag list to match new view

What
----

In conjunction with the creation of a new custom ZenDesk view for the team we add in a few "category" topics.

How to review
-------------

- Check for typos
- Check page renders OK

Who can review
--------------

not @schmie
